modules/ossm-config-otel.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/traces/ossm-distr-tracing-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel_{context}"]
+= Configuring {OTELName} with Service Mesh
+
+You can integrate {SMProductName} with {OTELName} to instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior.
+
+.Prerequisites
+
+* A TempoStack is installed and configured as described here: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/distributed_tracing/distributed-tracing-platform-tempo#installing-a-tempostack-instance[Installing a TempoStack instance].
+* An instance of an `OpenTelemetry` collector is already configured in the `istio-system` namespace.
+* An Istio instance is created in the `Istio` resource (`Istio.spec.profile`).
+* An Istio CNI instance is created in the `Istio` resource (`Istio.spec.profile`).
+
+.Procedure
+
+. Enable tracing in {SMProductName} in `spec.values.meshConfig`:
+// Configure {SMProductName} to enable tracing, and define the {OTELShortName} tracing providers in your `meshConfig`:
++
+.Example `spec.values.meshConfig` for tracing
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  ...
+  name: default
+spec:
+  namespace: istio-system
+  ...
+  values:
+    meshConfig:
+      enableTracing: true
+----
+
+. Define the {OTELShortName} tracing providers in `spec.values.meshConfig.extensionProviders`:
++
+.Example `spec.values.meshConfig.extensionProviders`
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  ...
+  name: default
+spec:
+  namespace: istio-system
+  ...
+  values:
+    meshConfig:
+      enableTracing: true
+      extensionProviders:
+      - name: otel-tracing
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local #<1>
+----
```

Comment:
I would combine these two steps into one. Otherwise the user has to edit the same object twice in succession.